### ISSUE TITLE
Error after first trial completes and when showing preview window in inspector.

### ISSUE THREAD

**Issue by svizcay** [BioMotionLab/TUX#4]
I'm using bmlTUX 0.17.0
textmesh pro preview 8 (as with 13 the gui didn't look the same as intended).

When I click "preview design" from the inspector or when I complete the first trial (return key pressed), I get an error saying:
```
ArgumentOutOfRangeException: Non-negative number required
Parameter name: totalWidth
```
and
```
GUI Error: You are pushing more GUIClips than you are popping. Make sure they are balanced.
```
I attach the backtrace

Thanks a lot for this framework.
[bmltux_errors.txt](https://github.com/BioMotionLab/TUX/files/4689311/bmltux_errors.txt)

**Comment by AdamBebko** [comment]
Thanks for the report I'll try to figure out what may be happening. Could you clarify whether this is in the tutorial or a custom project please?

**Comment by AdamBebko** [comment]
Fixed in v0.17.1

Thanks so much. It was related to string formatting for the preview system. 

To update your package, in unity package manager, remove the package, then type in the git repository with the new version number (0.17.1).
